packages/venia-ui/lib/components/CartPage/PriceSummary/taxSummary.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import gql from 'fraql';
+import { Price } from '@magento/peregrine';
+
+const DEFAULT_AMOUNT = {
+    currency: 'USD',
+    value: 0
+};
+
+/**
+ * Reduces applied tax amounts into a single amount.
+ *
+ * @param {Array} applied_taxes
+ */
+const getEstimatedTax = (applied_taxes = []) => {
+    if (!applied_taxes.length) {
+        return DEFAULT_AMOUNT;
+    } else {
+        return {
+            currency: applied_taxes[0].amount.currency,
+            value: applied_taxes.reduce((acc, tax) => acc + tax.amount.value, 0)
+        };
+    }
+};
+
+/**
+ * A component that renders the discount summary line item.
+ *
+ * @param {Object} props.classes
+ * @param {Object} props.data query response data
+ */
+const TaxSummary = props => {
+    const { classes } = props;
+
+    const tax = getEstimatedTax(props.data);
```

Comment:
Should this not be `const tax = getEstimatedTax(props.data.applied_taxes);`?

Reply:
`data` is already the destructured value from the query response. I didn't do the destructuring within the line item components because they don't really know how to obtain their data from a larger object -- they only expect to receive the data in the fragment that they have asked for.